engine/app/components/citizens_advice_components/footer.rb

```diff
@@ -7,7 +7,7 @@ class Footer < Base
     renders_one :feedback_link, "FooterFeedbackLink"
 
     renders_one :legal_summary, lambda { |text|
-      text.presence || t("citizens_advice_components.footer.legal_summary")
```

Comment:
This PR is mainly trying to resolve instances where a legal summary is set but empty. At the moment they wouldn't get the updated office move text.